common/api/src/main/java/com/dfsek/terra/api/util/MathUtil.java

```diff
@@ -39,10 +86,10 @@ public static double standardDeviation(List<Number> numArray) {
         double mean = sum / length;
         
         for(Number num : numArray) {
-            standardDeviation += FastMath.pow2(num.doubleValue() - mean);
+            standardDeviation += Math.pow(num.doubleValue() - mean, 2);
```

Comment:
Use
```java
double distanceToMean = num.doubleValue() - mean;
standardDeviation += distanceToMean * distanceToMean;
```
instead of `Math.pow(num.doubleValue() - mean, 2)`.

Reply:
see above comments on Math.pow